HelloWorlds/graphql/lib/models/user.js

```diff
@@ -0,0 +1,40 @@
+const mongoose = require('mongoose');
+
+const {
+  LanguagesSchema
+} = require('./languages');
+
+const UserSchema = new mongoose.Schema({
+  name: String,
+  // standard types
+  age: {
+    type: Number,
+    index: true
+  },
+  ln: {
+    type: [LanguagesSchema],
+    // you may include other schemas (here included as array of embedded documents)
+    default: [],
+    alias: 'languages' // in schema `ln` will be named as `languages`
+
+  },
+  contacts: {
+    // another mongoose way for providing embedded documents
+    email: String,
+    phones: [String] // array of strings
+
+  },
+  gender: {
+    // enum field with values
+    type: String,
+    enum: ['male', 'female']
+  },
+  someMixed: {
```

Comment:
what will `someMixed` be used for? could we remove it and then add it if needed in the future?